src/routes/auth.route.js

```diff
@@ -0,0 +1,21 @@
+import express from 'express';
+import { authController } from '../controllers/auth.controller.js';
+import { catchError } from '../catchError.js';
+
+export const authRouter = new express.Router();
```

Comment:
The use of `new` is not necessary when creating an instance of `express.Router()`. You can simply use `express.Router()`.